### DISCUSSION TITLE
Welcome to books Discussions!

### DISCUSSION THREAD

**Discussion by rmehta** [frappe/books#212] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by buddudevelopers** [comment]
is there an addon framework + market place for frappe books or plans to introduce one?

**Comment by nutcracker53** [comment]
I have installed Books ver 0.31.0. How do I activate the POS app. Is there a separate app for this.? Where could I get this..Thank you

**Comment by KBAccountant** [comment]
Hello, I run theaccounting department for a small compnay that does internationl business. I found this via an internet search. I have developed a farily comprehensive CoA (Chart of Accouts) that I would love to not only share, but also learn if this is something that can be imported. I am also very, very new to Github and am not a programmer by any stretch. I do see a way to attach the file, which I have done. Any guidence would be appreciated. Thanks. Ken
[Comp COA.csv](https://github.com/user-attachments/files/22052243/Comp.COA.csv)

**Reply by Altrix-One** [reply]
Hi @KBAccountant have you looked at ERPNext and their accounting module ? You are welcome to contact me for more info.
